src/Components/List.js

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react"
+import { AiFillStar, AiOutlineStar, AiFillDelete } from "react-icons/ai";
+import { TiTick } from "react-icons/ti"
+import { RxCross2 } from "react-icons/rx"
+
+export default function List(props) {
```

Comment:
destructure props

Reply:
Bob zeroll taught me not to do that. It's easy to understand which one is taken as props.

Reply:
but i still did it for u.